src/app/pie-graph/page.tsx

```diff
@@ -6,55 +6,65 @@ import { HTMLControl } from "@/components/Docs/Control/components/HTMLControl/HT
 import { Control } from "@/components/Docs/Control/Control";
 import { ComponentProps, useState } from "react";
 import { Legend } from "@/components/Legend/Legend";
+import { Segment } from "../../utils/graph/graphUtils";
+import { RollTheDice } from "@/components/RollTheDice/RollTheDice";
 
 export default function Page() {
-	const [pie, setPie] = useState<ComponentProps<typeof Pie>>({
+	const [pie, setPie] = useState<ComponentProps<typeof PieGraph>>({
 		loading: false,
 		donut: false,
 		labels: true,
-		children: "",
 	});
-	const setPiePartial = (partial: Partial<ComponentProps<typeof Pie>>) => setPie((prev) => ({ ...prev, ...partial }));
+	const setPiePartial = (partial: Partial<ComponentProps<typeof PieGraph>>) => setPie((prev) => ({ ...prev, ...partial }));
+
+	const [data, setData] = useState<Segment[]>(MOCK_DATA);
+
+	const handleRandomizedData = (newData: Segment[]) => {
```

Comment:
`handleRandomizedData` -> `onRollClick` 

to stick with the convention of naming event handler functions `on<Name>` instead of `handle<Name>` throughout project.

May seem small but it's important to make the codebase read as if it was written by 1 person even though it's written by 15+

May publish these in a documentation file somewhere